docs/glossary.md

```diff
@@ -0,0 +1,5 @@
+# Glossary
+
+* **Native**: By something native, we mention something living on the starknet network (eg: native ETH = StarknetETH)
+* **KEVM**: By something KEVM, we mention something living on the kakarot contract but with additional admin properties managed by the kakarot contract (eg: KEVM WETH)
+* **EVM**: By something EVM, we mention something living on the kakarot contract (eg: an EVM token)
```

Comment:
EVM tokens don't live on the Kakarot contract per se:

As of December 2022, they are being deployed on Starknet (through Kakarot) as smart contracts that store their own EVM bytecode. 
They are later on called using an address registry (a separate smart contract holding a 1 to 1 mapping of EVM contracts' EVM addresses <-> starknet addresses). Only Kakarot can interact with this registry contract.